proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,64 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime inorder for SDL Core to get accurate system time information which is utilized by the TLS handshake mechanism.

```

Comment:
I think in order should be two words here... can we change to the text below?
This proposal is about adding a new RPC GetSystemTime in order for SDL Core to get accurate system time information which is utilized by the TLS handshake mechanism.

Reply:
@mrapitis, Fixed in https://github.com/smartdevicelink/sdl_evolution/pull/324/commits/bbe0f4151e53cc0c528ca66fba27f1371e87423f